packages/devtools/cli/src/commands/app/open.ts

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import { Args, Flags, ux } from '@oclif/core';
+import chalk from 'chalk';
+import { chromium } from 'playwright';
```

Comment:
seems like this could be a heavy dependency and it could slow down CLI startup time. I would do a dynamic import instead in the `run` function of the command